sycl/include/sycl/ext/intel/esimd/detail/emu/esimd_emu_functions_v1.h

```diff
@@ -0,0 +1,46 @@
+//==----- esimd_emu_functions_v1.h - DPC++ Explicit SIMD API ---------==//
```

Comment:
here and in all other esimd files: correct location is `sycl/ext/intel/experimental/esimd/emu/detail`

Reply:
Same, it was done like that:
```
Copy
include/CL/sycl/ONEAPI to sycl/ext/oneapi and include/CL/sycl/INTEL to
sycl/ext/intel as is.
```
What do you propose for esimd? Don't touch esimd code in include/CL/sycl/INTEL (except marking namespace as deprecated)?

Reply:
INTEL/esimd.h is already OK (maybe the `#include elementary` should be moved to the redirected to header.
```cpp
#pragma once

#include <CL/sycl/detail/defines_elementary.hpp>

__SYCL_WARNING("CL/sycl/INTEL/esimd.hpp usage is deprecated, include "
               "sycl/ext/intel/experimental/esimd.hpp instead")

#include <sycl/ext/intel/experimental/esimd.hpp>
```

`INTEL/esimd/detail/emu` should be moved to `sycl/ext/intel/experimental/esimd/emu/detail` for now. We need to review namespace in esimdcpu_device_interface.h and move further if needed. @dongkyunahn-intel - FYI.

Reply:
Done

Reply:
I already have changes for moving files in the `INTEL/esimd/detail/emu` in my PR for ESIMD_CPU support - https://github.com/intel/llvm/pull/4011/commits/7d78e713b496390fdca90755487430bf1cbcd374.

Reply:
I'll revert the above change in my PR when this PR is merged in the `origin/sycl` branch.